pages/report.tsx

```diff
@@ -83,6 +86,28 @@ export default function Report() {
   const { session } = useSessionUser();
   const defaultFormFormValues = useInitialFormValues();
   const submitForm = useSubmitSharkSightingForm();
+  const [ImageFileUuid, setImageFileUuid] = useState<string>();
+
+  const onFileChange = async (e: React.ChangeEvent<HTMLInputElement>) => {
+    const fileList = e.target.files;
+    console.log(fileList);
+    if (!fileList) return;
+    for (let i = 0; i < fileList.length; i++) {
+      if (!fileList[i]) return;
+      const uuid = uuidv4();
+      try {
+        const { data, error } = await supabase.storage
+          .from("user-report-images")
+          .upload(uuid + ".png", fileList[i], {
```

Comment:
Guessing this is temporary, but these images might not be `png`s. So we'll want to look at the file's `name` and split off the file extension instead of doing this.